src/manifest.ts

```diff
@@ -46,22 +47,40 @@ export async function writeManifest(manifest: Manifest) {
 
 /**
  * Returns true if the manifest is valid.
+ */
+export async function isValidManifest(): Promise<boolean> {
+  return await getManifest() != null;
+}
+
+/**
+ * Ensures the manifest is correct for running a function.
+ * The manifest must include:
+ * "executionApi": {
+ *   "access": "MYSELF"
+ * }
+ */
+export async function isValidRunManifest(): Promise<boolean> {
+  if (isValidManifest) {
```

Comment:
Shouldn't you be running this function? Doesn't this current like just check if the function is defined?

Reply:
Yes, great catch, should be `if (isValidManifest()) {`. I'll look at adding another test for this as well.

Reply:
And that function returns a promise, so I think you'll also need to await the promise to get the actual boolean result.